tests/test_advanced.py

```diff
@@ -394,30 +395,34 @@ def test_positive_satellite_repositories_setup(setup_subscribe_to_cdn_dogfood, a
             for repo in sat_repos[ver]:
                 assert repo in result["stdout"]
 
+    # 6.12 till not GA
+    contacted = ansible_module.shell(Advanced.run_repositories_setup({"version": "6.12"}))
```

Comment:
If there is no special need to use both, I would suggest to use just one of the `command`/`shell` ansible module.

Reply:
Preferably `command` but I know sometimes we need to use `shell` instead.

Reply:
yes, Using command module, we cannot use variables like $HOSTNAME and operations like  ">",  "|",  ";" ,  etc with shell we can.

Reply:
@vsedmik Good catch, update this step to use the command module, and for beta repos part we need shell module for export command and ";"